CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+# Contributing to this project/repository
+To contribute is fairly easy:
+
+Until a proper RISC OS git tool is being released, we encourage you to use for your "local git" either a Windows computer or a Linux computer or Virtual Machine.
+
+There are some very useful tips [here at the ROOL Forum](https://www.riscosopen.org/content/documents/git-cheatsheet)
+
+## General guidelines
+In general you'd want to:
+- Git clone a repo from here to your local git
+- Create your branch using git branch (usually for branching naming you'd want to create the branch name as: <your nick>-<what type of changes you are applying>)
+- When done you have to submit your branch for review requesting a code review to the Code Reviews Team
+- If you can please make sure your commit messages are clear and meaningful to help reviewers to understand what you were trying to achieve
+- When possible try to make small commits so that the review process can be quicker
+- Make sure you keep the documentation up-to-date with your changes
+- Where applicable stick to RISC OS Style guide
+- Make sure you've tested your changes before submitting a Pull Request
```

Comment:
```suggestion
- Git clone a repo from here to your local git
- Create your branch using git branch (we suggest that you name your branches following this pattern: <your nick>-<what type of changes you are applying>)
- Make sure your commit messages are clear and meaningful to help reviewers to understand what you were trying to achieve
- When possible try to make small commits and Pull Requests so that the review process can be quicker
- Make sure you keep the documentation up-to-date with your changes
- Where applicable stick to RISC OS Style guide
- Make sure you've tested your changes before submitting a Pull Request
- Submit your branch for review when it is done, do this by making a Pull Request and requesting a review from the Code Reviews Team
```